README.md

```diff
@@ -292,6 +297,8 @@ aws-adfs integrates with:
                                       factors. For "Passcode" and "WebAuthn
                                       Security Key" factors, it is always "None".
       --enforce-role-arn              Only allow the role passed in by --role-arn.
+      --verification-code TEXT        Verification code for multi-factor
```

Comment:
I'd probably rename this option and/or update its description to make it clear it only works for Azure AD MFA:
```suggestion
      --aad-verification-code TEXT        Verification code for Azure AD multi-factor authentication
```

Reply:
Comitted. But we will now have to read in that option in the code though. Will make the required changes.